src/issue-activity.ts

```diff
@@ -35,32 +32,11 @@ export class IssueActivity {
   linkedReviews: Review[] = [];
 
   async init() {
-    function fn<T>(func: () => Promise<T>) {
-      return func();
-    }
-    const decoratedFn = retryAsyncUntilDefinedDecorator(fn, {
-      delay: this._configuration.delayMs,
-      maxTry: this._configuration.maxAttempts,
-      async onError(error) {
-        try {
-          const content = "Failed to retrieve activity. Retrying...";
```

Comment:
Since you removed this, what happens on retry, is any comment posted saying the plugin is retrying?

Reply:
No, but I think it doesn't make sense to post on every retry because if it's works after 10 retries then it's all good but if it fails after 10 retries I would like to know that.